snap/local/etc/cruise-control/log4j.properties

```diff
@@ -0,0 +1,60 @@
+rootLogger.level=INFO
+appenders=console, kafkaCruiseControlAppender, operationAppender, requestAppender
+
+property.filename=${sys:cruisecontrol.logs.dir}
```

Comment:
**niptick** This does not seem a system property as [these](https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html). I checked [here](https://logging.apache.org/log4j/3.x/manual/configuration.html#PropertySubstitution) and "main" seems possibly more appropriate. 

Kafka does not use any, so maybe we can use the same. It would definitely be good to have a consistent UX here across. On this, we could actually allow log level tuning using the same property `charmed.kafka.log.level`

Reply:
Ah, I didn't realise there were more labels than just `sys`. I'll find a better option, probably `main`.

I have a suspicion that Kafka does something different with the configuration file for backwards compatibility, as it didn't work without this syntax. I can see if Kafka works with it also.

And yes, agreed on log-level, but again it wasn't working so I've left it out, but it's on the todo list for later!

Reply:
Oh, that's interesting if it did not work. Are they (kafka and CC) using the same logging library? I suppose the variable resolution is handled there. If this syntax works on Kafka, let's use this one instead (possibly with main) and add the log level here as well, just to keep consistency